cli-activemq/src/main/java/com/redhat/mqe/aoc/Main.java

```diff
@@ -48,7 +48,7 @@
         ConnectionManagerFactory bindConnectionManagerFactory(AocConnectionManagerFactory f);
 
         @Binds
-        MessageFormatter bindMessageFormatter(OpenwireMessageFormatter f);
+        JmsMessageFormatter bindMessageFormatter(OpenwireJmsMessageFormatter f);
```

Comment:
Neviem ako funuguju Bindy :(

Reply:
Ja uz taky pomalu ne, od ty doby co jsem to tam daval... :( Je to normalni DI framework, akorat ze vsechno se dela v generovanem kodu misto dynamicky za behu (jak to dela Guice a dalsi DI frameworky).

Nastesti na to neni potreba moc sahat.

Tohle konkretne znamena neco jako ze kdyz mam vytvorit objekt, jehoz konstruktor chce parametr typu JmsMessageFormatter, tak se ma vytvorit instance tridy OpenwireJmsMessageFormatter a ta se strci do toho konstruktoru.

Reply:
Aha, cool :)